src/solver/type_getters.jl

```diff
@@ -93,6 +97,7 @@ function get_atmos(config::AtmosConfig, params)
         edmfx_model,
         precip_model,
         cloud_model,
+        noneq_mode = implicit_noneq_moist ? Implicit() : Explicit(),
```

Comment:
`noneq` may refer to many things. Maybe something more precise like `cloud_formation_sources`?